src/main/java/com/ays/location/model/entity/LocationEntity.java

```diff
@@ -0,0 +1,48 @@
+package com.ays.location.model.entity;
+
+import com.ays.common.model.entity.BaseEntity;
+import com.ays.user.model.entity.UserEntity;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Point;
+
+/**
+ * Location entity, which holds the information regarding location.
+ */
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "AYS_USER_LOCATION")
+public class LocationEntity extends BaseEntity {
```

Comment:
Bu lokasyon kısmı doğrudan User'a ait bir obje olduğu için `user.location` paketi altına taşıyıp, adını da `UserLocationEntity` olarak adlandırmamız daha doğru olur.

Reply:
location package altında olması daha iyi olur. Diğer türlü herşeyi admin ve user ana package'lerin altına alacağız.